MIGRATION.md

```diff
@@ -0,0 +1,114 @@
+# Migrating from `dp-frontend-renderer` to `dp-renderer`
+
+## Migrating assets and models
+
+The frontend service is responsible for setting up the assets binary and source path.
+
+You should also store all assets and models within the application itself with the following structure:
+
+```md
+.
+├── asset                    # relevant templates & localisations from dp-frontend-renderer
+│   ├── templates          
+│   ├── localisations  
+|   |   ├──  service.en.toml
+|   └── └──  service.cy.toml
+└── model                    # relevant models from dp-frontend-models
+```
+
+## Updating the Makefile
+
+In order for `dp-renderer` to work correctly once the assets have been migrated over, we use `go-bindata` to generate a combined assets source file.
```

Comment:
```suggestion
For `dp-renderer` to work correctly once the assets have been migrated over, we use `go-bindata` to generate a combined assets source file.
```

Not at all a requirement to change and nothing wrong here. But, I really like concise wording when it comes to documentation.